assets/course-theme/contact-teacher.js

```diff
@@ -0,0 +1,72 @@
+/**
```

Comment:
This file is responsible for making the message sending to teacher without refreshing the page and showing the success message the way it is in Course Theme (Learner|Focus Mode) design.